src/app/modules/recipe-list/components/recipe-carousel.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, HostListener, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-recipe-carousel',
+  templateUrl: './recipe-carousel.component.html',
+  styleUrls: ['./recipe-carousel.component.scss']
+})
+export class RecipeCarouselComponent {
+
+  public limit = 3;
+  images = [
+    {path: 'https://www.kwestiasmaku.com/sites/v123.kwestiasmaku.com/files/bigos_z_kiszonej_kapusty_01_0.jpg'},
+    {path: 'https://www.zajadam.pl/wp-content/uploads/2010/10/rosol-przepis-tradycyjny-6504345.jpg'},
+    {path: 'https://1.bp.blogspot.com/-uLQuV_Nv7pg/XgDjRFSWsoI/AAAAAAAAT9c/tkM2CPXDomgInv25hXF9VsbozIgmhdAuQCNcBGAsYHQ/s640/79703750_451330148869455_7787295256956895232_n.jpg'},
+    {path: 'https://d1uz88p17r663j.cloudfront.net/resized/2020_04_03T14_00_59_image_944_531.ashx'},
+    {path: 'https://www.mojegotowanie.pl/media/cache/default_view/uploads/media/recipe/0001/100/gulasz-wieprzowy-z-szynki.jpeg'},
+    {path: 'https://kuron.com.pl/wp-content/uploads/2018/04/P4124119.jpg'},
+    {path: 'https://ocdn.eu/pulscms-transforms/1/Et5k9kpTURBXy8zYTg1NGQ5YTUwNmQ3MGUzNWVmM2E2N2FmOGJiNWM2Yy5qcGeTlQMAzKLNFEDNC2STBc0DFM0BvJMJpjZiZjgyYgaBoTAB/pierogi-z-kapusta.jpg'}
+  ]
+
+  @HostListener('window:resize', ['$event'])
+  onResize(event: { target: { innerWidth: number; }; }) {
+    if (event.target.innerWidth < 768) {
+      this.limit = 1;
+    } else {
+      this.limit = 3;
+    }
```

Comment:
Create enum for screen breakpoints:

```
export enum BreakpointsEnum {
  xs = 576,
  sm = 768,
  md = 992,
  lg = 1200
}
```

```suggestion
    this.limit = event.target.innerWidth < BreakpointsEnum.sm ? 1 : 3;
```